istio/src/main/java/com/alibaba/nacos/istio/mcp/NacosToMcpResources.java

```diff
@@ -48,71 +46,78 @@
  */
 @org.springframework.stereotype.Service
 public class NacosToMcpResources {
-    
+
```

Comment:
Please do not change the indent

Please use nacos code style to reformat codes.

Reply:
Thanks for reminding, it's done